src/components/DetailViewHeader.jsx

```diff
@@ -110,20 +131,39 @@ export class DetailViewHeader extends React.PureComponent {
                 canManageUsers={true}
               />
             )}
-            <Button className={classes.exportButton} variant="contained" color="primary">
-              <a
-                target="_blank"
-                rel="noopener noreferrer"
-                href={`${terraUrl}/#import-data?url=${window.location.origin}&snapshotId=${of.id}&snapshotName=${of.name}&format=snapshot`}
+            <Dialog open={isOpen}>
+              <DialogContent>
+                <Typography variant="h5">Your data snapshot is being exported</Typography>
+                {/* TODO: Make this loading state more descriptive */}
+                <div className={clsx(classes.centered, classes.content)}>
+                  <CircularProgress />
+                </div>
+              </DialogContent>
+            </Dialog>
+            <Tooltip
+              title="Exporting a snapshot to a workspace means that all members of your workspace will be able to have read only access to the tables and files in the snapshot"
+              arrow
+            >
+              <Button
+                onClick={this.exportToWorkspaceCopy}
+                className={classes.exportButton}
+                variant="contained"
+                color="primary"
               >
                 Export to Workspace
-              </a>
-            </Button>
+              </Button>
+            </Tooltip>
           </Card>
         </Grid>
       </Grid>
     );
   }
 }
 
-export default withStyles(styles)(DetailViewHeader);
+function mapStateToProps(state) {
+  return {
+    isOpen: state.snapshots.dialogIsOpen,
```

Comment:
You should use a different variable in the state.  This one if for the snapshot creation dialog.  I'd create an exportDialogIsOpen method to track it separately